MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -411,25 +661,46 @@ int main(int argc, char * argv[])
                 {
                     if (!(EndsWith(unpackDestination, L".vhd") || (EndsWith(unpackDestination, L".vhdx"))))
                     {
+                        errorCode = msixmgr::GetErrorCodeFromHRESULT(E_INVALIDARG);
+                        errorDesc = L"Invalid VHD file name. File name must have .vhd or .vhdx file extension. HRESULT " + errorCode + L".";
+
                         std::wcout << std::endl;
                         std::wcout << "Invalid VHD file name. File name must have .vhd or .vhdx file extension" << std::endl;
                         std::wcout << std::endl;
+
+                        // Telemetry : Workflow Log
+                        QueryPerformanceCounter(&msixMgrLoad_EndCounter);
+                        workflowElapsedTime = msixmgr::CalcWorkflowElapsedTime(msixMgrLoad_StartCounter, msixMgrLoad_EndCounter, msixMgrLoad_Frequency);
+                        msixmgr::TraceLogWorkflow(workflowId.c_str(), cli.GetOperationTypeAsString().c_str(), false, workflowElapsedTime, errorCode.c_str(), errorDesc.c_str());
+
                         return E_INVALIDARG;
                     }
                     else
                     {
                         if (cli.GetVHDSize() == 0)
                         {
+                            errorCode = msixmgr::GetErrorCodeFromHRESULT(E_INVALIDARG);
+                            errorDesc = L"VHD size was not specified. Please provide a vhd size in MB using the -vhdSize option. HRESULT " + errorCode + L".";

```

Comment:
Same as above

Reply:
As discussed, all the Telemetry variables will have the same Error Message as the actual Console statements. The change for outputting the Error Message on Console from a variable will be done as part of Error Message Improvement Task that will be taken later.